### ISSUE TITLE
Quiver version incompatible when using flutter current release

### ISSUE THREAD

**Issue by osdiab** [DirectMyFile/github.dart#108]
With `github.dart@^3.0.0`, you get this error:

```
[mobile_app] flutter packages get
Running "flutter packages get" in mobile_app...
Because every version of flutter_test from sdk depends on quiver 0.29.0+1 and github >=2.1.4 depends on quiver >=0.20.0 <0.25.0, flutter_test from sdk is incompatible with github >=2.1.4.
So, because gitizen_mobile_app depends on both github ^3.0.0 and flutter_test any from sdk, version solving failed.
pub get failed (1)
exit code 1
```

I'm overriding it like so:

```
# pubspec.yml
dependency_overrides:
  quiver: ^0.29.0
```

but I'd prefer not to. Is there a reason newer versions of quiver are restricted?

**Comment by osdiab** [comment]
Looks like it's fixed in 4.0.0-dev, though would be nice if I could still use the current stable

**Comment by robbecker-wf** [comment]
@osdiab Can you check if this issue is resolved for you with the latest release (4 or 4.0.1) ?

**Comment by robbecker-wf** [comment]
Closing ... please reach out if this is still a problem.
